Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -113,27 +113,32 @@ protected void setSyncVarHookGuard(ulong dirtyBit, bool value)
         /// </summary>
         protected readonly List<SyncObject> syncObjects = new List<SyncObject>();
 
-        /// <summary>
-        /// NetworkIdentity component caching for easier access
-        /// </summary>
-        NetworkIdentity netIdentityCache;
-
         /// <summary>
         /// Returns the NetworkIdentity of this object
         /// </summary>
```

Comment:
this doc comment needs to be  directly above `public NetworkIdentity netIdentity`